conf/logback.loki.xml

```diff
@@ -13,6 +13,20 @@
             <sortByTime>true</sortByTime>
         </format>
     </appender>
+    <appender name="AUDIT" class="com.github.loki4j.logback.Loki4jAppender">
+        <http>
+            <url>http://localhost:3200/loki/api/v1/push</url>
```

Comment:
Can you make a .local config if you are going to commit localhost:3200

Reply:
ppppffftt this is supposed to be loki:3200